### DISCUSSION TITLE
problem with Selected and Managed mode buttons

### DISCUSSION THREAD

**Discussion by HappySimmer** [flybywiresim/aircraft#2885] [General]
Hi there
If I change from managed mode to selected all is fine, but if I want to go back to managed mode the button doesn't work anymore.
This goes for the SPEED en HDG/NAV.  Funnily enough with ALT the problem doesn't occur
Today I installed the latest developer mod and the problem is still there.  Please advise me

**Comment by Inaxair** [comment]
Can you provide the route, the sim version, the aircraft version, the flight phase and more details? 

Thanks
Inaxair from FlyByWire Simulations
